tile_generator/helm.py

```diff
@@ -0,0 +1,41 @@
+import os
+import sys
+import yaml
+import json
+
+def find_required_images(values):
+    images = []
+    values = { k.lower():v for k,v in values.items() }
+    for key, value in values.items():
+        if key == 'image':
+            if isinstance(value, dict):
+                image = value.get('repository')
+                tag = value.get('tag', value.get('imagetag', None))
+            else:
+                image = value
+                tag = values.get('tag', values.get('imagetag', None))
+            if tag is not None:
+                image += ':' + tag
+            images += [ image ]
+        else:
+            if isinstance(value, dict):
+                images += find_required_images(value)
+    return images
+
+def get_chart_info(chart_dir):
+    chart_file = os.path.join(chart_dir, 'Chart.yaml')
```

Comment:
do we expect the `Chart.yml` to always have to be capitalized?

Reply:
That's what the helm spec says, but if/when we discover exceptions we will add code for them